Include/cpython/unicodeobject.h

```diff
@@ -445,6 +437,14 @@ enum PyUnicode_Kind {
         (0xffffU) :                                                     \
         (0x10ffffU)))))
 
+Py_DEPRECATED(3.3)
+static inline Py_ssize_t _PyUnicode_WSTR_LENGTH_impl(PyObject *op) {
```

Comment:
nitpick: I suggest to remove `_impl` suffix, the `_` prefix is enough ;-)

Reply:
`_PyUnicode_WSTR_LENGTH` macro is used in unicodeobject.c.
Using same name will make confusion.

Reply:
> _PyUnicode_WSTR_LENGTH macro is used in unicodeobject.c.

That's not an issue in practice: the macro override the static inline function. It has the priority. No?

If I should pick which file uses the worst names, I pick the C file. I prefer to have the best names for the public C API.

I used "_impl" suffix for _PyObject_GC_TRACK() macro which calls _PyObject_GC_TRACK_impl(). One year later, I started to see "_PyObject_GC_TRACK_impl" names in backtraces in bug reports. I'm not satisfied to "leak" the "_impl" suffix to users. But I failed to find a better name for this one.

Reply:
> That's not an issue in practice: the macro override the static inline function. It has the priority. No?

Note that unicodeobject.c uses both of `_PyUnicode_WSTR_LENGTH` and `PyUnicode_WSTR_LENGTH`, and they have different semantics.

If `_PyUnicode_WSTR_LENGTH` inline function is overridden by macro,
`PyUnicode_WSTR_LENGTH` will be broken.